stable/weave-scope/requirements.yaml

```diff
@@ -0,0 +1,10 @@
+# We alias the child chart names to avoid having to deal with hyphenated 
+# YAML keys.
+
```

Comment:
This file is unnecessary. It is used for dynamically linked dependencies only, whereas you added the dependencies directly to the charts folder. I guess that's the reason why you can't use aliases in this case.

Reply:
Maybe that's it.  There's some discussion related to aliases going on in kubernetes/helm#2993.  IMHO it should be possible to alias directly-added subcharts, but for now since I'm not actually using the aliases I can remove this.

Reply:
Actually I just realized there is a reason to still have this file -- I'm using conditionals to allow for disabling deployment of either of the subcharts.  I deleted the "alias" lines though.